migration/v0.19.x-v0.20.x/remove-grafana-ops.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -eo pipefail
+
+./bin/ck8s ops helmfile sc -f helmfile -l app=grafana-ops delete
```

Comment:
Don't use delete, use destroy
```suggestion
./bin/ck8s ops helmfile sc -f helmfile -l app=grafana-ops destroy
```